app/main/forms/cancel.py

```diff
@@ -0,0 +1,22 @@
+from flask.ext.wtf import Form
+from wtforms import RadioField, validators
+
+
+class CancelBriefForm(Form):
```

Comment:
Curious - what was behind the decision to use WTF instead of content loader in the end? Was it validation/errors?

Reply:
Sooo, basically we had a content loader version set up, working and ready to go. However, because design wants a single page question thing we couldn't pass the 'question' as the question attribute of the content loader YAML to the standard template because the standard template wouldn't allow for the bigger title. No problem, we just won't pass that from the YAML. We'll pass it as a heading.
Well apparently not because in order to pass tests every YAML question has to have 'question' attribute (seemingly arbitrarily).
By this point it got to feeling like we were trying to force a square peg through a round hole and it would be cleaner not to mess too heavily with the content loader and just WTForm it.

Reply:
Good to know. Worth documenting on the content loader README that it's not suitable for this sort of situation? (Or yet another tech debt ticket to introduce the ability to skip questions for tests if there's a header instead?)

Reply:
Other reasons why WTF is a good choice
- probably the quicker of the two ways to build
- The information that will be sent to the API is not a big dict of JSON data (content loader is good for this). Instead, it is just parsing one key and deciding which API endpoint to call
- We do not need to validate this using a JSON schema